### DISCUSSION TITLE
Does anyone know if I can give this internship certificate to my college ?

### DISCUSSION THREAD

**Discussion by s-aksh** [github/india#563] [Students]
It is mandatory in my college for a final year student to do a four week internship.
Does anyone know if I can give this internship certificate to my college ?

**Comment by HemantSachdeva** [comment]
Yes why not?

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official GitHub Externship Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏
